components/src/core/components/Chart/PieChart.vue

```diff
@@ -289,4 +280,4 @@ export default defineComponent({
 });
 </script>
 
-<style src="./pie-chart.scss" lang="scss" scoped></style>
+<style src="./pie-chart.scss" lang="scss"></style>
```

Comment:
removed scoping here to support styles for custom tooltip generated. but scoped within class name `oxd-pie-chart`